models.py

```diff
@@ -111,6 +144,17 @@ def get_consistent_temporal_coverage(self) -> bool:
 
         return end > start
 
+    def get_temporal_coverage_range(self) -> str | None:
+        coverage = self.payload["temporal_coverage"]
+        if coverage:
+            start = coverage.get("start") or "?"
+            end = coverage.get("end") or "?"
+            return f"{start} - {end}"
+
+    def get_spatial_coordinates(self) -> str | None:
+        if coords := ((self.payload.get("spatial") or {}).get("geom") or {}).get("coordinates"):
+            return repr(coords)
```

Comment:
How will we show this? This can be a very long string.

Maybe we want to compute statistics (number of pairs, bbox...) instead of the full payload?

If we need the full payload for further analysis, shouldn't we store it as JSON in database?

Reply:
Right now we're displaying the very long string as is.
I agree we could do better, but I'd do it in another PR. It's not obvious to me what stats are best to flag abnormal spatial.

Reply:
https://github.com/ecolabdata/ecospheres-dashboard-backend/issues/58